fastly/validators_test.go

```diff
@@ -78,30 +78,170 @@ func TestValidateLoggingPlacement(t *testing.T) {
 	}
 }
 
+func TestValidateDirectorQuorum(t *testing.T) {
```

Comment:
Just to follow up on the conversations.  Suggestion to turn these into table driven tests.  Allowing increased feedback on all test scenarios instead of stopping at the failing point in the test data array.  Placing comment here, but same suggestion for all the of the test functions in the validator_test.go file.